andela/titan-dev-ah-frontend
# #163518655 Add following functionality on readArticle page

#### What does this PR do?

- It adds the ability to follow an author
#### Description of Task to be completed?

- Added the following action to the `follow` button
- Implemented the functionality of clicking twice to unfollow

#### How should this be manually tested?
- After cloning the repository, check out `ft-follow-another-user-163518655` branch
- Run `yarn` to install the project dependencies
- Run `yarn start` to start the application then, you navigate to any of the articles
- There you shall see the follow button next to the author's info of that article
- If you're already following that author, click to unfollow and vice-versa

#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #163518655
